native/proto/src/proto/operator.proto

```diff
@@ -46,12 +46,19 @@ message Operator {
   }
 }
 
+message Statistics {
```

Comment:
Nice! 
For reference (Spark): [Statistics.scala](https://github.com/apache/spark/blob/b8060e585873348793b38de06288fb49fbe24708/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/Statistics.scala#L54)